elements/rh-card/demo/variants.html

```diff
@@ -156,20 +158,25 @@ <h2 slot="header">Card with slotted image header. Full width image.</h2>
     }
   }
 
-  rh-card.full {
+  rh-card.image {
     &::part(header) {
       margin: 0;
     }
 
-    & :is(h1, h2, h3, h4, h5, h6)[slot="header"] {
-      padding-inline: var(--rh-space-xl, 24px);
-    }
-
     & img {
       display: block;
       min-width: 100%;
       height: auto;
     }
+
+    &.heading {
+      & h2 {
+        font-weight: var(--rh-font-weight-heading-bold, 700);
```

Comment:
Font weight for headers should be `--rh-font-weight-heading-medium`. Not sure if it matters in the demo, but it might confuse users to see a bolder-than-usual heading.